internal/commands/waypoint/actions/create.go

```diff
@@ -137,7 +191,28 @@ func createAction(c *cmd.Command, args []string, opts *CreateOpts) error {
 		return fmt.Errorf("failed to create action %q: %w", opts.Name, err)
 	}
 
-	fmt.Fprintf(opts.IO.Err(), "Action %q created.", opts.Name)
+	// Choose fields based on action flavor
+	var fields []format.Field
+	if opts.Request.Agent != nil {
```

Comment:
Instead of using the fields of the request here, could we instead keep the response from `WaypointServiceCreateActionConfig`? That would show what the server actually returned instead of showing to the user what they input for the request via the flag. In theory those should be the same, but if there were ever a server issue where something wasn't correctly applied in the request it'd be nice to know that here.

Reply:
Good call, done!

Reply:
Whoops that broke the tests but they're fixed now if you can take one more look 😅 Thanks!